Source/OxyPlot/Series/BarSeries/RectangleBarSeries.cs

```diff
@@ -152,15 +157,25 @@ public override void Render(IRenderContext rc)
                 return;
             }
 
-            var clippingRect = this.GetClippingRect();
+			// determine render range
+			var xmin = this.XAxis.ActualMinimum;
+			var xmax = this.XAxis.ActualMaximum;
+			this.winIndex = FindBarIndex(Items, xmin, this.winIndex);
+
+	        if (this.winIndex > 0)
+	        {
+		        this.winIndex--;
+	        }
 
-            int i = 0;
+			var clippingRect = this.GetClippingRect();
```

Comment:
please create a separate branch for changes to the RectangleBarSeries.
Also, create a separate issue for this change.
This is performance optimizations, right?